Makefile

```diff
@@ -0,0 +1,37 @@
+
+BIN:=iso-chooser-menu
+
+URL:=http://cdimage.ubuntu.com/streams/v1/com.ubuntu.cdimage.daily:ubuntu-server.json
+
+CFLAGS:=-Wall -Werror -Wfatal-errors -std=c11
```

Comment:
We shouldn't really override the default CFLAGS here I think. I don't speak Makefile well enough any more to know what the best practice here is though.

Reply:
Done.
https://www.gnu.org/software/make/manual/html_node/Appending.html was helpful.
Using the system CFLAGS exposed usage of undefined behavior, which I fixed.